### ISSUE TITLE
RichTextBoxFormatBar: Default font, style etc.

### ISSUE THREAD

**Issue by ApDAV** [xceedsoftware/wpftoolkit#1415]
Is there a way to set the default font, size, color, etc for RichTextBoxFormatBar so when the pop up appears they are in those specified defaults.

**Comment by XceedBoucherS** [comment]
No, because when the RichTextFormatBar is loaded, it will set its FontFamily, Bold, Color....based on the selected text.
